Makefile

```diff
@@ -132,7 +132,7 @@ tooldep:
 	mtools --version
 
 test: tooldep
-	$(GO) test -v $(PKGS)
+	$(GO) test -v ./...
```

Comment:
So `PKGS = $(shell go list ./... | tr '\n' ',' | head -c -1)` is just overcomplicated version of simple `./...` ?

Reply:
Also `PKGS` seems to be not used anywhere else so I think it may be removed.